generated/nidaqmx/_library_interpreter.py

```diff
@@ -2232,11 +2232,50 @@ def get_ai_chan_cal_exp_date(self, task, channel_name):
 
     def get_analog_power_up_states(
             self, device_name, channel_name, channel_type):
-        raise NotImplementedError
+        state = []
+
+        args = [device_name]
+        argtypes = [ctypes_byte_str]
+
+        for index in range(len(channel_name)):
+            state_element = ctypes.c_double
```

Comment:
you need to call the ctor here

```suggestion
            state_element = ctypes.c_double()
```

Reply:
I have updated this as per suggestions